web-client/src/components/validation/Validation.vue

```diff
@@ -24,60 +24,122 @@
       </div>
     </div>
     <div v-show="validationStore.step === 0">
-      <div
-        v-if="memoryStore.models.size > 0"
-        class="grid gris-cols-1 md:grid-cols-2 lg:grid-cols-3 items-stretch gap-8 mt-8"
-      >
-        <div
-          v-for="[path, metadata] in memoryStore.models"
-          :key="path"
-          class="contents"
-        >
-          <ButtonCard
-            :click="() => selectModel(path)"
-            :button-placement="'left'"
-          >
+      <div class="flex flex-col gap-16">
+        <div v-if="memoryStore.models.size > 0">
+          <IconCard>
             <template #title>
-              {{ taskTitle(metadata.taskID) }}
+              Model Library — <span class="italic">Downloaded and Ready to Test</span>
```

Comment:
should not be called 'downloaded'. they grow and stay locally. `locally available and ready to test`?